beacon-chain/core/transition/transition.go

```diff
@@ -258,20 +258,19 @@ func ProcessSlots(ctx context.Context, state state.BeaconState, slot types.Slot)
 			return nil, errors.Wrap(err, "could not process slot")
 		}
 		if time.CanProcessEpoch(state) {
-			switch state.Version() {
-			case version.Phase0:
+			if state.Version() == version.Phase0 {
```

Comment:
is it possible to continue using switch statements here ? 
```go
switch {
case state.version() == version.Phase0:
// do something
case state.version() >= version.Altair:
// do another thing
default:
// return error
}
```
It seems easier to read compared to multiple else-ifs

Reply:
You and @potuz conflict with each other : )
https://github.com/prysmaticlabs/prysm/pull/11825#discussion_r1056901482
https://github.com/prysmaticlabs/prysm/pull/11825/commits/c082176a9e00107eb90ed83852db8e20e1392a6f

I was team switch but later "switched" to team if-else because there likely wont be more than 2 cases to cover